### ISSUE TITLE
Should this CoC be more of a democracy?

### ISSUE THREAD

**Issue by falkTX** [ContributorCovenant/contributor_covenant#597]
I am opening this issue to start a discussion, see tittle.

From what I understood, as it stands right now, the CoC is basically managed by @CoralineAda which is free to close issues anytime, sometimes not giving a chance to reply back.
(feel free to correct me though)

I cannot see how this can have the best of the community in mind.
If a single person (or even a small group of people) are the single owners of such an important document, personal agenda and beliefs will eventually be visible in the project. (we are all people in the end, which get corrupted and have deep feelings about stuff we care about)

Since we want to have the best for the community, shouldn't the community also have a say?

PS: @CoralineAda no need for a reply right away, I do not want to pressure you more than what you already have in hands. I just would like to have a small discussion about this.

**Comment by beoran** [comment]
No, this project does not need to be a democracy, and as was stated before by it's maintainers and rightful owners, it is not. 

But, due to the open source nature of this project, if you feel the content of the CoC should be decided democratically by vote of other open source project maintainers, you are free to set up a fork of this CoC within a project where decisions *are* made democratically. 

That is likely to be a more productive approach than to keep asking the maintainers of this project to change the goals of this project.

**Comment by Atavic** [comment]
@falkTX As a country with a self-appointed leader, this repository seems like a Dictatorship.

**Comment by beoran** [comment]
Many good open source projects have a benevolent dictator. From the point of view that those one who works the most on a project has the most property rights to it, this is often quite justified. Less justified is when this CoC is used to try and remove said rightful owner...

**Comment by falkTX** [comment]
We just had #602, closed without any word said about it.
In a democracy, everyone has a right to speak, which I guess is not what the creators of this project have in mind. It just looks bad for them...
I had hopes for this CoC, but now I am just getting disappointed.

Their mentality is basically "Do as I say, not as I do".
If you can't live by your own rules, whose rules are you going to follow? how are we going to trust you?
I don't think I can take it seriously anymore.

**Comment by beoran** [comment]
Be that as it may. 
Will to power, I would say. 
Don't get disappointed. Fork. 
It's the open source way to work.

**Comment by SilicaAndPina** [comment]
it should be a meritocracy.

**Comment by camerons** [comment]
I'm happy to work on the ContributorCovenant under a benevolent dictator model. I trust @CoralineAda's motivations and competency based on the quality of her work to date, and her significant evangelising.
However, I too would respectfully suggest that a more sustainable long term strategy would be to adopt "Project Steering Committee" and decision making process, as you see in the many Open Source projects that are adopting the ContributorCovenant. I'd suggest that such a Project Steering Committee be kick started by @CoralineAda inviting specific active members of this community that she trusts. (Maybe 2 to 10 extra people to start with). Of all communities, I suspect this is the most likely to attract a diverse community.
If you need help with grunt work for drafting such documents, feel free to lean on me (I've drafted such documents for projects within the Open Source Geospatial Foundation before).

**Comment by jonschlinkert** [comment]
It already is a democracy. 

- Anyone who agrees with it can use it, as is
- Anyone who has suggestions for improvements can do so through a pull request or discussions on issues
- Anyone who disagrees can fork the project or create a new one with entirely new language

**Comment by Mitchell128134** [comment]
That's not a democracy. A democracy is everyone has a say, not where everyone can ask the person in charge to do something. I think @CoralineAda should be removed due to her repeated violations of her own rules. Also, she'll shut down any dissenting issues before they have a chance to be seen.

**Comment by OceanPond** [comment]
Democracy is people railing against absolute power in search of ... absolute power. 🙈🙉🙊

**Comment by Aspie96** [comment]
To anybody interested in #602, it was immediately locked (not closed) and then promptly removed, but you can easily read it here:
https://web.archive.org/web/20181001094604/https://github.com/ContributorCovenant/contributor_covenant/issues/602

(Notice: The maintainer cannot ask Archive to remove the page, as it does not break copyright law).

**Comment by camerons** [comment]
@Aspie96 #602 publicly calls out for a specific person to be removed due to comments that they have made outside of this forum.
While I feel it is appropriate to call out a behaviour type as inappropriate, getting personal about it is likely to lead to unconstructive personal attracts.
I appreciate the irony of a rule not holding true for the creator of the rule, but the world is a messy place and humans are way too complicated to define by a set of rules. I think this should be used as input into improving future versions of the ContributorConvenant rather than to throw out a valuable member of our community on a technicality. I suggest it is best to recognise that we are all flawed, and a better way to solve human problems is to start by helping improve a person's behaviour rather than firing them.

**Comment by Aspie96** [comment]
>  @Aspie96 <https://github.com/Aspie96> #602 publicly calls out for a
specific person to be removed due to comments that they have made outside
of this forum.
She has done that as well!

Also, the Code of Conduct applies outside of the project as well for
maintainers, thus she broke the CoC.

However, we are OT (my fault). I posted the Archive version simply because
the issue was mentioned. In case someone was reading the conversation, it
is useful for them to have a reference.

Il giorno dom 5 gen 2020 alle ore 20:34 Cameron Shorter <
notifications@github.com> ha scritto:
